modules/ztp-troubleshooting-the-managed-cluster.adoc

```diff
@@ -0,0 +1,73 @@
+// Module included in the following assemblies:
+//
+// *scalability_and_performance/ztp-zero-touch-provisioning.adoc
+
+[id="ztp-troubleshooting-the-managed-cluster_{context}"]
+= Troubleshooting the managed cluster
+
+.Procedure
+
+. Check the status of the managed cluster:
++
+[source,terminal]
+----
+$ oc get managedcluster
+----
++
+.Example output
+[source,terminal]
+----
+NAME            HUB ACCEPTED   MANAGED CLUSTER URLS   JOINED   AVAILABLE   AGE
+SNO-cluster     true                                   True     True      2d19h
+----
++
+If the status in the `AVAILABLE` column is `True`, the managed cluster is being managed by the hub.
++
+If the status in the `AVAILABLE` column is `Unknown`, the managed cluster is not being managed by the hub.
+Use the following steps to continue checking to get more information.
+
+
+. Check the AgentClusterInstall install status:
++
+[source,terminal]
+----
+$ oc get clusterdeployment -n <cluster-name>
+----
++
+.Example output
+[source,terminal]
+----
+NAME        PLATFORM            REGION   CLUSTERTYPE   INSTALLED    INFRAID    VERSION  POWERSTATE AGE
+Sno0026    agent-baremetal                               false                          Initialized
+2d14h
+----
++
+If the status in the `INSTALLED` column is `false`, the installation was unsuccessful.
+
+. You can find information about why the installation was unsuccessful by describing the
+AgentClusterInstall resource and checking the status and event logs:
++
+[source,terminal]
+----
+$ oc describe agentclusterinstall -n <cluster-name> <cluster-name>
+----
+
+. If possible, fix the errors and do the following:
```

Comment:
How do you fix the errors? Maybe you need another step, then I suggest making this step more descriptive, perhaps like:
```suggestion
. After you resolve the errors, reset the cluster:
```